### PR TITLE
Add tags to subscribers

### PR DESCRIPTION
**Pull request by jawadakram20**

### CODE REVIEW

**Comment by jawadakram20** [comment]
Can you please review my Pr and merge this as soon as possible because i am stuck on adding tags.

File: lib/aweber/resources/subscriber.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class Subscriber < Resource
       api_attr :status,        :writable => true
       api_attr :custom_fields, :writable => true
       api_attr :ad_tracking,   :writable => true
+      api_attr :tags,   :writable => true
```
Comment (amberheilman) [inline lib/aweber/resources/subscriber.rb]: nit: align `:writable => true` to the right

**Comment by jawadakram20** [comment]
@amberheilman I have fixed PR.Can you please merge it now.Thanks

**Comment by jawadakram20** [comment]
@amberheilman No updates yet.Can you please update the gem...Thanks

**Comment by amberheilman** [comment]
This change will not allow you to create a subscriber with tags. It will return a signature mismatch because it is not properly json encoding the array.

Example:
```ruby
new_subscriber = {}
new_subscriber['email'] = 'email@email.com'
new_subscriber['name'] = 'willy wonka'
new_subscriber['tags'] = ['tag1', 'tag2', 'tag3']
aweber.account.lists[list_id].subscribers.create(new_subscriber)
---------------------------------------------
Invalid signature (AWeber::CreationError)
```
You will need to add this to /lib/aweber/collection.rb
```ruby
    def create(attrs={})
      params = attrs.merge("ws.op" => "create")

      if params.has_key?('custom_fields')
        if params['custom_fields'].is_a?(Hash)
            params['custom_fields'] = params['custom_fields'].to_json
        end
      end
      if params.has_key?('tags')
        if params['tags'].is_a?(Array)
            params['tags'] = params['tags'].to_json
        end
      end
```

**Comment by amberheilman** [comment]
You will not be able to retrieve a subscribers tags with this library, given that the client traverses the collection or uses the find_by method; both of which do not have tags in the json response. The subscriber entry lookup will need to be added in order to have a symmetric POST/GET for the tags field.

**Comment by jawadakram20** [comment]
@amberheilman why cant we just add this method to the gem and let users to create the tags through gem instead of writing their own lib changes.I am currently using this changes and tags are created successfully.But to_json change will do that perfectly.

**Comment by RajRoR** [comment]
:+1: @jawadakram20 

@amberheilman Do we have any updates regarding merge of this PR?

**Comment by amberheilman** [comment]
Comments have not been addressed.

**Pull request closed without merging**